data/bootstrap_seed.yaml

```diff
@@ -0,0 +1,171 @@
+---
+webvirt: 192.168.1.1
+site_name: sitename
+maas_api_key: consumerkey:tokenkey:tokensecret
+maas_uuid: nonce
```

Comment:
Not sure what uuid is in this context or why drydock needs this as part of the hardware toplogy.